lib/screenplay/outfront_takeover_tool/alerts/reminders.ex

```diff
@@ -22,7 +22,7 @@ defmodule Screenplay.Alerts.Reminders do
     schedule_work()
 
     if url == "" do
-      Logger.info("No Slack Webhook URL found")
+      # Logger.info("No Slack Webhook URL found")
```

Comment:
Any reason for commenting this out?

Reply:
Oops, that shouldn't have been committed. Log message was just getting in the way when debugging so I meant for that to be temporary.